### DISCUSSION TITLE
Problem with tags plugin - folder can't be created

### DISCUSSION THREAD

**Discussion by Creat1veM1nd** [squidfunk/mkdocs-material#7501] [Q&A] [answered]
Hello, 

I added the built-in tags plugin as stated in the documentation but when added, my site does not start. Turned out that there is a problem with a folder called "aux" which obviously shall be created but windows does not allow folders with that name. With that, the folder can not be created at all. (not sure if that restriction is bound to system drive or in general, I do not have any other drives on my notebook than c)

In addition, I also get this message: 
```
WARNING  -  Config value: 'plugins'. Warning: ('tags_file', 'Unrecognised configuration name: tags_file')
```

I configured it this way
```
plugins:
  - tags:
      tags_file: tags.md
```

Is there any chance to have a workaround ? So far I did not found any option to change that folder's name but maybe I overlooked it ?

Thanks in advance for any help!

**Comment by kamilkrzyskow** [comment, marked as answer]
Hello @Creat1veM1nd,
the tags plugin shouldn't need any `aux` directory, as for the `tags_file` it's a supported config option:
https://github.com/squidfunk/mkdocs-material/blob/d95e0e62def5068ec8003d378c164ab5b752e21c/material/plugins/tags/config.py#L33-L35

So please prepare a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/). To make sure it works with the newest Material for MkDocs version and a minimal setup.

**Reply by Creat1veM1nd** [reply]
Thanks for your quick response!

In order to prepare the reproduction I also reinstalled material (as I did before starting with my project today) and gave it another try. It now worked perfectly. I guess the installation earlier did not work as expected, sadly I did not have any logs of it - so no idea, what exactly went wrong. 

Now, all is fine. Thanks for your help!

**Reply by kamilkrzyskow** [reply]
No worries about not keeping logs, in general users need to use the newest version of the theme to get the latest fixes, even if the issues were present 3 versions ago, as versions aren't updated retroactively.
So it's good to have it confirmed, if anybody will come across a similar issue, they just have to install the newest version.
Thanks for the cooperation 🤝
